Libs/vtkSegmentationCore/vtkOrientedImageDataResample.h

```diff
@@ -160,6 +164,23 @@ class vtkSegmentationCore_EXPORT vtkOrientedImageDataResample : public vtkObject
   ///   mask is non zero
   static bool ApplyImageMask(vtkOrientedImageData* input, vtkOrientedImageData* mask, double fillValue, bool notMask = false);
 
+  /// Get the labelmap values contained within the mask
+  /// \param binaryLabelmap Input image to get values from
```

Comment:
I made the change, but I've had a thought.

These functions are not necessarily labelmap-specific, even though they've been developed for (and probably will only be used) in that context.
In that case, would it be worth making them more generic?

Reply:
What exactly do you have in mind? Could you give examples of methods that we have now and we would have if they were more generic?

Reply:
I have nothing in mind. It's just that most of the functions in vtkOrientedImageDataResample work on both labelmap and non-labelmap volumes.

Reply:
I agree that it is good if we can find a name that can be interpreted more generally, but the name should be specific enough so that you can guess what it means.

Reply:
OK, I see, the question is about calling it a labelValue or just value (can be int or float).

Since equality check for float values is very fragile, I would avoid using non-integer types for arguments that rely on equality check. Checking if larger/smaller (for example for thresholds) is safe.